env_test.go

```diff
@@ -118,3 +119,56 @@ func TestLoadConfigFileNotConfigured(t *testing.T) {
 		t.Errorf("got unexpected error: %#v", err)
 	}
 }
+
+func TestWriteConfigFile(t *testing.T) {
+	dir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(dir)
+	os.Setenv("HOME", dir)
+	expectedContent := "some content"
+	err = writeConfigFile(bytes.NewReader([]byte(expectedContent)))
+	if err != nil {
+		t.Fatal(err)
+	}
+	fullPath := filepath.Join(dir, ".tranor", "config.json")
```

Comment:
It seems `github.com/tsuru/tsuru/cmd` has a function called JoinWithUserDir. Any reason not to use it?